### DISCUSSION TITLE
How to handle complex merged cells in tables

### DISCUSSION THREAD

**Discussion by aneeqahsan01** [PaddlePaddle/PaddleOCR#13420] [Q&A]
Hi, I have a pdf with complex merged cells and would like to know if there is some script (post processing) or any other means to handle this. 
An example is this in pdf:
![table_example](https://github.com/user-attachments/assets/3aae43d3-3f0b-45c2-af83-95f01b10da24)

**Comment by aneeqahsan01** [comment]
@UserWangZz @jzhang533

**Comment by GreatV** [comment]
please refer to: https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/docs/quickstart_en.md#pp-structure-quick-start

**Reply by aneeqahsan01** [reply]
That does not help a lot, The result is not a table just recognition of entities and texts. Here's the result:
![result](https://github.com/user-attachments/assets/6cfc71cf-54f5-424d-9034-6fa805dc6686)


 I started exploring this: https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/table/README.md
But the results are very bad.

**Reply by UserWangZz** [reply]
you need use your data finetune the table recognition model.

**Reply by saloni644c** [reply]
Hey, can you elaborate on finetune part? I am having the same issues with such tables
